control/control.c

```diff
@@ -699,6 +701,19 @@ main(int argc, char *argv[])
 				INFO("device csr written to %s", dev_csr_file);
 			}
 		} break;
+		case DAEMON_TO_CONTROLLER__CODE__RESPONSE: {
+			if (!resp->has_response)
```

Comment:
protobuf_free_message?

Reply:
is done after switch case block line 706:721